src/seedu/addressbook/commands/CommandResult.java

```diff
@@ -21,6 +21,14 @@ public CommandResult(String feedbackToUser) {
         relevantPersons = null;
     }
 
+    
+    /**
+     * This returns feedbackToUser.
```

Comment:
The start of the header comment does not follow the convention (i.e. first sentence should be something like `Selects next line …` instead of `Select next line` or `This method is for selecting …`)